Library/Formula/fpc.rb

```diff
@@ -17,10 +15,25 @@ class Fpc < Formula
     sha1 "1476a19ad7f901868fcbe3dc49e6d46a5865f722"
   end
 
+  if MacOS.version >= :yosemite
+    # Use updated crt1 linking logic from upstream
+    patch do
+      url "https://github.com/graemeg/freepascal/commit/74a5a86cc7f1f978404f9dc172655658ce18c2d1.diff"
+      sha1 "97fdcee78faf11bca12fc654eb399f92941d3910"
+    end
+    # Manually set minimum OS X version for bootstrapping (note '1099' due to length limit)
+    patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission in a comment in the formula file. Thanks!